src/components/checkbox/index.tsx

```diff
@@ -1,56 +0,0 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { Checkbox } from "antd";
-import "../../App.css";
-interface CheckboxProps {
-  checkboxNumber: number;
-}
-
-const CustomCheckbox = ({ checkboxNumber }: CheckboxProps) => {
-  const [letters, setLetters] = useState<string[]>([]);
-  const [checkvalue, setCheckValue] = useState<boolean>(false);
-  const url =
-    "https://navirego-interview-mc3narrsb-volodymyr-matselyukh.vercel.app/api/letters/";
-  useEffect(() => {
-    const handleFetch = async () => {
-      const res = await axios.get(`${url}${checkboxNumber}`);
```

Comment:
Axios should not be called directly. The reason we don't call axios directly because the configuration of the axios should be consistent across every API call. 
fetchUtils.ts or http.ts file should ideally store the axiosInstance code.